compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -1451,6 +1451,9 @@ bitflags! {
         const IS_LINEAR          = 1 << 3;
         // If true, don't expose any niche to type's context.
         const HIDE_NICHE         = 1 << 4;
+        // If true, the struct's layout can be randomized using
+        // the seed stored in [`ReprOptions.layout_seed`]
```

Comment:
Does `.` work there? As opposed to `::`. Actually, this is not a doc comment, so the `[...]` syntax does nothing.

Reply:
Yah, it's not a doc comment but I just did it by habit